backends/dpdk/dpdkAsmOpt.cpp

```diff
@@ -355,22 +355,100 @@ CopyPropagationAndElimination::copyPropAndDeadCodeElim(
         } else if (auto jc = stmt->to<IR::DpdkJmpLessStatement>()) {
             instr.push_back(new IR::DpdkJmpLessStatement(jc->label,
                         replaceIfCopy(jc->src1, false), replaceIfCopy(jc->src2)));
+        } else if (auto jc = stmt->to<IR::DpdkJmpLessOrEqualStatement>()) {
```

Comment:
this implementation now has a lot of redundancy, I suspect you could do much better by using a visitor with a preorder rmethod on the base classes. But this will work.

Reply:
I would like to create a new PR for refactoring.